src/directive.js

```diff
@@ -187,6 +188,24 @@ function getLinkFunction($http, theme, util, type) {
                     chart.showLoading({ text: scope.config.errorMsg || '没有数据', textStyle: textStyle });
                 }
             }
+            if(type == 'gauge2') {
+                var imgEle = element.find('img')[0];
+                if(!imgEle) {
+                    imgEle = new Image();
+                    imgEle.style.width = '80px';
```

Comment:
possible to make image size relative to parent element? also the whole gauge chart. user can customize the size by adding style attributes on the directive itself.